java-tests/validating/src/test/kotlin/io/spine/test/options/ValidateConstraintTest.kt

```diff
@@ -67,8 +68,8 @@ internal class ValidateConstraintTest {
         val violations = error.get().constraintViolationList
         violations.size shouldBe 2
 
-        violations[0].fieldPath.getFieldName(0) shouldBe "first_line"
-        violations[1].fieldPath.getFieldName(0) shouldBe "town"
+        val failedFields = violations.map { it.fieldPath.getFieldName(0) }
+        failedFields shouldContainExactlyInAnyOrder listOf("first_line", "town")
```

Comment:
This is because `OptionGenerator`-provided `if`s are rendered AFTER the `Rule`-provided ones in `validate(...)`.
Thus, affecting the ordering of violations. 

The emerged question is the following: do we provide ordering expectations regarding the produced violations? So, if fields with ordinal numbers `#4, #8, #9` fail, we get violations for `#4, #8, #9` in the same order.

For example, `Rule` framework renders constraints according to the order of declaration in the original .proto file. So, if `myField` is declared first and has three validation options, the first three `if` blocks in `validate(...)` will be asserting exactly the value of `myField()` getter. As for now, it is not true for `OptionGenerator`.

Reply:
@yevhenii-nadtochii we have briefly discussed the matter locally, and we think the ordering of violations does not matter. At least, as of now. We may want to tweak it in the future, but for now it does not matter.